tasks/task907_dialogre_identify_relationships.json

```diff
@@ -0,0 +1,439 @@
+{
+    "Contributors": [
+        "Matthew Huff"
+    ],
+    "Source": [
+        "DialogRE"
+    ],
+    "Categories": [
+        "Identification"
+    ],
+    "Definition": "In this task, you are given a dialog between 2 or more individuals. Within the dialog, there will be clues that indicate two individuals were or are dating. You will be prompted at the end to answer which Speaker (or named individual) is dating which other Speaker (or named individual).",
```

Comment:
The "definition" must say what we expect in the output (which seems to be "1" or "2"?)